SHT20 Temperature and Humidity Sensor

rob-vaughn opened the pull request:
This pull request adds the Sensirion SHT20 I2C Temperature and Humidity sensor, built into a waterproof probe case, to WipperSnapper_Components.

https://www.dfrobot.com/product-1636.html

brentru commented:
Hi @rob-vaughn  - are you also planning to add an I2C Component driver to WipperSnapper (Instructions via the guide here: https://learn.adafruit.com/how-to-add-a-new-component-to-adafruit-io-wippersnapper/adding-an-i2c-component-driver)?

rob-vaughn commented:
Hi Brent,

I don't have a ton of experience coding these sensors, but I can give it a
shot.

I work with the Canadian government doing climate change research and am
really interested in the Adafruit IO and its capabilities. Specifically,
I'd like to add a number of sensors that I use regularly in my research
with Adafruit boards. The SHT20 sensor that I added to the component list
is one of them. Is there any support I can access to get these sensors
added to WipperSnapper?

Thanks!

On Tue., Nov. 8, 2022, 12:23 p.m. Brent Rubell, ***@***.***>
wrote:

> Hi @rob-vaughn <https://github.com/rob-vaughn> - are you also planning to
> add an I2C Component driver to WipperSnapper (Instructions via the guide
> here:
> https://learn.adafruit.com/how-to-add-a-new-component-to-adafruit-io-wippersnapper/adding-an-i2c-component-driver
> )?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1307483533>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEJJQVIWJMONWSUO4X3WHJ467ANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

brentru commented:
@tyeth  - any interest in supporting the SHT20 linked above?

rob-vaughn commented:
Thanks Brent.

What is the general protocol here? Are sensors added mostly by other people
or do you guys have an ongoing list of sensors that you're loading into
WipperSnapper?

On Tue., Nov. 8, 2022, 12:49 p.m. Brent Rubell, ***@***.***>
wrote:

> @tyeth <https://github.com/tyeth> - any interest in supporting the SHT20
> linked above?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1307519949>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEKJG6RGANNCZEKHC33WHKADLANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

brentru commented:
@rob-vaughn  Sensors on IO are added by me (I work for Adafruit on WipperSnapper, IO, etc..) and external contributors. This repository's [issues page](https://github.com/adafruit/Wippersnapper_Components/issues) lists ongoing sensors that we'd/users would like added to WipperSnapper. They are not in any particular order to allow people to pick them up + contribute sensors. There's no ETA. I am currently working on Features for IO/WipperSnapper (such as - NeoPixel, DotStar, PWM) and will wrap back around to adding sensors from this list when I complete those tasks.

We have a guide on adding the driver code for these sensors here: https://learn.adafruit.com/how-to-add-a-new-component-to-adafruit-io-wippersnapper/adding-an-i2c-component-driver

If you get stuck at any point - submit a pull request and I'll be able to review and help triage it with you.

@tyeth is a community member who recently added a few SHTx sensor family member sensors to IO. I tagged them as I was curious if they'd be interested in adding a WipperSnapper driver code to support the SHT20.

tyeth commented:
Hiya! You're project / work sounds great, and let me say the wippersnapper thing makes it so quick and easy and exactly why I was adding all the sensors I had. Sadly I've got distracted programming on the esp32's and trying to find a job here in the UK that isn't just programming, maybe Canada's where it's at😄 
 I took a quick look and I couldn't see an official sensirion library, although [dfrobot have one](https://github.com/DFRobot/DFRobot_SHT20), but worth noting @brentru this next one I stumbled upon has a nice simple VPD (#71) calculation there 😉  https://github.com/u-fire/uFire_SHT20/blob/master/src/uFire_SHT20.cpp#L87

I had a quick go and it's totally untested, but if you want to try it  @rob-vaughn  then you can follow the guide for developing wippersnapper, as mentioned by Brent: https://github.com/tyeth/Adafruit_Wippersnapper_Arduino/compare/main...add_DFRobot_SHT20
 -- The guide is basically clone the repository ```git clone https://github.com/tyeth/Adafruit_Wippersnapper_Arduino``` then you'll be on git branch 'main' which is the same as adafruit repository, and then install all the libraries listed in library.properties and then load ```examples\wippersnapper_demo\wippersnapper_demo.ino```, pick your board in arduino (you'll need ESP32 boards added if using them) then try and build (compile) and the flash (upload) the software onto the board. If that works then you have wippersnapper building fine, at that point change to my new branch in git with ```git checkout add_DFRobot_SHT20``` then build the arduino file again and upload again to test the new sensor.
  It might be worth trying the basic example from the DFRobot product wiki page too. Let us know how you get on or how much / little hand holding to offer.

rob-vaughn commented:
Thanks for all your help! I just realized that the sht20 sensor I have is
the predecessor of the superior sht30 sensor (which I also have), and
it has the exact same application. I guess this means that adding the sht20
would be redundant.

Would you ever be adding a CO2 sensor? A computer programmer friend of mine
programmed an esp8266 to read the Senseair K30 sensor (
https://senseair.com/products/flexibility-counts/k30/) for one of our
experiments. It's i2c, relatively cheap and quite accurate.

Cheers!

On Tue., Nov. 8, 2022, 5:55 p.m. Tyeth Gundry, ***@***.***>
wrote:

> Hiya! You're project / work sounds great, and let me say the wippersnapper
> thing makes it so quick and easy and exactly why I was adding all the
> sensors I had. Sadly I've got distracted programming on the esp32's and
> trying to find a job here in the UK that isn't just programming, maybe
> Canada's where it's at😄
> I took a quick look and I couldn't see an official sensirion library,
> although dfrobot have one <https://github.com/DFRobot/DFRobot_SHT20>, but
> worth noting @brentru <https://github.com/brentru> this next one I
> stumbled upon has a nice simple VPD (#71
> <https://github.com/adafruit/Wippersnapper_Components/pull/71>)
> calculation there 😉
> https://github.com/u-fire/uFire_SHT20/blob/master/src/uFire_SHT20.cpp#L87
>
> I had a quick go and it's totally untested, but if you want to try it
> @rob-vaughn <https://github.com/rob-vaughn> then you can follow the guide
> for developing wippersnapper, as mentioned by Brent:
> ***@***.***_DFRobot_SHT20
> <https://github.com/tyeth/Adafruit_Wippersnapper_Arduino/compare/main...add_DFRobot_SHT20>
> -- The guide is basically clone the repository git clone
> https://github.com/tyeth/Adafruit_Wippersnapper_Arduino then you'll be on
> git branch 'main' which is the same as adafruit repository, and then
> install all the libraries listed in library.properties and then load
> examples\wippersnapper_demo\wippersnapper_demo.ino, pick your board in
> arduino (you'll need ESP32 boards added if using them) then try and build
> (compile) and the flash (upload) the software onto the board. If that works
> then you have wippersnapper building fine, at that point change to my new
> branch in git with git checkout add_DFRobot_SHT20 then build the arduino
> file again and upload again to test the new sensor.
> It might be worth trying the basic example from the DFRobot product wiki
> page too. Let us know how you get on or how much / little hand holding to
> offer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1307878457>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEK7OEMGC7IXFQ7UEDDWHLD63ANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

brentru commented:
@rob-vaughn  We do support the scd40 which has a CO2 sensor.  You can add a new issue for the K30, I do not have one on-hand to add though.

tyeth commented:
Ahh don't let age rule them out, they still do the job, the DHT11/22's are seriously old (and unreliable) so worry not! I was surprised to read the SHT20's were lower power usage than the 3 series. I've got the 4's too. The second digit is 0/1/5 depending on accuracy+price.
 Regarding the CO2 sensors, I just read about the K30 recently in a comparison article of a new lower-power MH-Z13, I've got a load of Winsen MH-Z19b's which come in a few flavours but same driver code and are similar in quality to the K30's. Better than those are the Sensirion SCD30/40/41's which have recently been added. This was my purchase link for recent SCD41s that seem to be produced in china for laskakit.cz and have Qwiic connectors: https://www.aliexpress.com/item/1005004494224988.html 
I'd recommend getting the adafruit version, particularly if on the company dime, but I'm cheap and shipping/availability in the UK is a problem: 
SCD41 - no stock https://www.adafruit.com/product/5190
SCD40 - Stocked https://www.adafruit.com/product/5187

rob-vaughn commented:
You're solving all my problems. Thanks a lot!

On Thu., Nov. 10, 2022, 1:22 p.m. Tyeth Gundry, ***@***.***>
wrote:

> Ahh don't let age rule them out, they still do the job, the DHT11/22's are
> seriously old (and unreliable) so worry not! I was surprised to read the
> SHT20's were lower power usage than the 3 series. I've got the 4's too. The
> second digit is 0/1/5 depending on accuracy+price.
> Regarding the CO2 sensors, I just read about the K30 recently in a
> comparison article of a new lower-power MH-Z13, I've got a load of Winsen
> MH-Z19b's which come in a few flavours but same driver code and are similar
> in quality to the K30's. Better than those are the Sensirion SCD30/40/41's
> which have recently been added. This was my purchase link for recent SCD41s
> that seem to be produced in china for laskakit.cz and have Qwiic
> connectors: https://www.aliexpress.com/item/1005004494224988.html
> I'd recommend getting the adafruit version, particularly if on the company
> dime, but I'm cheap and shipping/availability in the UK is a problem:
> SCD41 - no stock https://www.adafruit.com/product/5190
> SCD40 - Stocked https://www.adafruit.com/product/5187
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1310632582>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEKV33V6S3FFGEY7GP3WHUVNTANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

brentru commented on components/i2c/sht20/definition.json:
```diff
@@ -0,0 +1,5 @@
+{
+  "displayName": "sht20",
+  "i2cAddresses": [ "0x40" ],
+  "subcomponents": [ "ambient-temp", "ambient-temp-celcius", "humidity" ]
```
@tyeth  @rob-vaughn So, `ambient-temp` is in celsius. Remove `ambient-temp-celcius. Use `ambient-temp-fahrenheit` instead

brentru commented:
@lorennorman  I'm not sure what's going on with this Action. Could you please check the schema error on this run's step: https://github.com/adafruit/Wippersnapper_Components/actions/runs/3415453348/jobs/5858112631

lorennorman commented:
Thanks for the nudge, @brentru!

The thing failing was the filename checker: non-Adafruit staff aren't allowed to modify some of the files in this repo, in this case the `**/schema.json` files.

Now, you might be saying to yourself, _"But we didn't modify those files in this PR?"_ and you'd be right, I was confused too at first. Then I realized that `main` has changed out from under us (big deploy this week that touched almost every file in order to add the `published: boolean` syntax.) Thus, the validator noticed the files in this PR were different from `main`.

The fix was to merge the `main` branch in, which I did, and those checks are now passing. The remaining validation failure is valid: `ambient-temp-celsius` is not a sensor type (or whatever.)

I'll improve the validator to have better messaging so that this situation is more understandable in the future, thanks all!

rob-vaughn commented:
Hey Brent and Tyeth,

Any plans to add a thermocouple amplifier?

https://www.adafruit.com/product/269

Thermocouples are really handy for measuring temps in all sorts of
scientific applications, especially when the temps are really high.

On Thu., Nov. 10, 2022, 1:22 p.m. Tyeth Gundry, ***@***.***>
wrote:

> Ahh don't let age rule them out, they still do the job, the DHT11/22's are
> seriously old (and unreliable) so worry not! I was surprised to read the
> SHT20's were lower power usage than the 3 series. I've got the 4's too. The
> second digit is 0/1/5 depending on accuracy+price.
> Regarding the CO2 sensors, I just read about the K30 recently in a
> comparison article of a new lower-power MH-Z13, I've got a load of Winsen
> MH-Z19b's which come in a few flavours but same driver code and are similar
> in quality to the K30's. Better than those are the Sensirion SCD30/40/41's
> which have recently been added. This was my purchase link for recent SCD41s
> that seem to be produced in china for laskakit.cz and have Qwiic
> connectors: https://www.aliexpress.com/item/1005004494224988.html
> I'd recommend getting the adafruit version, particularly if on the company
> dime, but I'm cheap and shipping/availability in the UK is a problem:
> SCD41 - no stock https://www.adafruit.com/product/5190
> SCD40 - Stocked https://www.adafruit.com/product/5187
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1310632582>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEKV33V6S3FFGEY7GP3WHUVNTANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

tyeth commented:
Sure that's mentally on their list, but stick in the request @rob-vaughn ❤️ I probably want the MCP9600 which is i2c and all the types after my quick googling. Not sure if that's got qwiic connectors at adafruit yet, sure as soon as it's revised they'll hoick it up the priority list (sparkfun have one).
<img width="660" alt="image" src="https://user-images.githubusercontent.com/6692083/208242781-2c47b6c5-e26d-430b-a8da-7abfb4c36e07.png">

 While you're at it can you update your pull request for this SHT20 component to use "ambiant" and "ambiant-faaaaarrrrenheit" or however you spell them instead of ambiant and _ambiant-celcius_ and then the checks in this issue will pass.
Happy holidays all.

rob-vaughn commented:
Will do, Tyeth.

So they won't prioritize it until Adafruit comes up with a MCP9600 board
that has a qwiic connect?

On Sat., Dec. 17, 2022, 8:55 a.m. Tyeth Gundry, ***@***.***>
wrote:

> Sure that's mentally on their list, but stick in the request @rob-vaughn
> <https://github.com/rob-vaughn> ❤️ I probably want the MCP9600 which is
> i2c and all the types after my quick googling. Not sure if that's got qwiic
> connectors at adafruit yet, sure as soon as it's revised they'll hoick it
> up the priority list (sparkfun have one).
> [image: image]
> <https://user-images.githubusercontent.com/6692083/208242781-2c47b6c5-e26d-430b-a8da-7abfb4c36e07.png>
>
> While you're at it can you update your pull request for this SHT20
> component to use "ambiant" and "ambiant-faaaaarrrrenheit" or however you
> spell them instead of ambiant and *ambiant-celcius* and then the checks
> in this issue will pass.
> Happy holidays all.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1356245865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEJUWQUE5KQ4NBMDHI3WNWZ45ANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

rob-vaughn commented:
Just changed the code to "ambient-temp-fahrenheit". Hopefully this resolves
it.

On Sat., Dec. 17, 2022, 8:55 a.m. Tyeth Gundry, ***@***.***>
wrote:

> Sure that's mentally on their list, but stick in the request @rob-vaughn
> <https://github.com/rob-vaughn> ❤️ I probably want the MCP9600 which is
> i2c and all the types after my quick googling. Not sure if that's got qwiic
> connectors at adafruit yet, sure as soon as it's revised they'll hoick it
> up the priority list (sparkfun have one).
> [image: image]
> <https://user-images.githubusercontent.com/6692083/208242781-2c47b6c5-e26d-430b-a8da-7abfb4c36e07.png>
>
> While you're at it can you update your pull request for this SHT20
> component to use "ambiant" and "ambiant-faaaaarrrrenheit" or however you
> spell them instead of ambiant and *ambiant-celcius* and then the checks
> in this issue will pass.
> Happy holidays all.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1356245865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEJUWQUE5KQ4NBMDHI3WNWZ45ANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

tyeth commented:
I don't know how priorities work, just that there is usually a buzz after each board gets re-spun with the qwiic connector. i.e. it often features in their Ask an Engineer as ladyada does the changes, and then once available it often appears as John Parks product of the week (up to 50% discount on tuesday evenings - max 10 items). The 3d printing brothers, Pedro and Noe Ruiz, also like to come up with some little project around the same time, which often includes a wippersnapper example.

In reality if you want it, you have a go rather than waiting on a wish and a prayer :) Most of the time Adafruit (or sparkfun etc) have already gone to the effort of making an arduino driver and example sketch so it's fairly doable if you have the time and inclination. 
 I think my copy and paste of a previous wippersnapper driver, then replacing with another adafruit sensor driver is probably half an hour, then a bit of back and forth with the github checks / CLang formating and can be live within the hour if your lucky, otherwise if there is feedback or issues then about 3-10days seems to be average. (If you add the component first like this pull request, then it can be approved as a in-development component, so you can test the driver you write straight away).

rob-vaughn commented:
I'll give it a go when I have a chance. Adafruit has the driver for the
MCP9600 online, so if I follow the tutorial for adding i2c drivers, it
shouldn't be too complicated.

Thanks again!

On Sun., Dec. 18, 2022, 1:29 p.m. Tyeth Gundry, ***@***.***>
wrote:

> I don't know how priorities work, just that there is usually a buzz after
> each board gets re-spun with the qwiic connector. i.e. it often features in
> their Ask an Engineer as ladyada does the changes, and then once available
> it often appears as John Parks product of the week (up to 50% discount on
> tuesday evenings - max 10 items). The 3d printing brothers, Pedro and Noe
> Ruiz, also like to come up with some little project around the same time,
> which often includes a wippersnapper example.
>
> In reality if you want it, you have a go rather than waiting on a wish and
> a prayer :) Most of the time Adafruit (or sparkfun etc) have already gone
> to the effort of making an arduino driver and example sketch so it's fairly
> doable if you have the time and inclination.
> I think my copy and paste of a previous wippersnapper driver, then
> replacing with another adafruit sensor driver is probably half an hour,
> then a bit of back and forth with the github checks / CLang formating and
> can be live within the hour if your lucky, otherwise if there is feedback
> or issues then about 3-10days seems to be average. (If you add the
> component first like this pull request, then it can be approved as a
> in-development component, so you can test the driver you write straight
> away).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1356842023>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEI3RZLBKKO344VNRMLWN5CZLANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

rob-vaughn commented:
Just found this online:

https://www.adafruit.com/product/5165?gclid=Cj0KCQiAtICdBhCLARIsALUBFcH0yal2_Vk987HMxA9e8NRVcHptusuSbfO-9xGyEGudReckt1GjnIoaAlbpEALw_wcB

Would this be a better option than the 9600 because of the qwiic connect? I
can't see why this board differs from the 9600 other than it having a qwiic
connect.

On Mon., Dec. 19, 2022, 6:31 a.m. Rob Vaughn, ***@***.***> wrote:

> I'll give it a go when I have a chance. Adafruit has the driver for the
> MCP9600 online, so if I follow the tutorial for adding i2c drivers, it
> shouldn't be too complicated.
>
> Thanks again!
>
> On Sun., Dec. 18, 2022, 1:29 p.m. Tyeth Gundry, ***@***.***>
> wrote:
>
>> I don't know how priorities work, just that there is usually a buzz after
>> each board gets re-spun with the qwiic connector. i.e. it often features in
>> their Ask an Engineer as ladyada does the changes, and then once available
>> it often appears as John Parks product of the week (up to 50% discount on
>> tuesday evenings - max 10 items). The 3d printing brothers, Pedro and Noe
>> Ruiz, also like to come up with some little project around the same time,
>> which often includes a wippersnapper example.
>>
>> In reality if you want it, you have a go rather than waiting on a wish
>> and a prayer :) Most of the time Adafruit (or sparkfun etc) have already
>> gone to the effort of making an arduino driver and example sketch so it's
>> fairly doable if you have the time and inclination.
>> I think my copy and paste of a previous wippersnapper driver, then
>> replacing with another adafruit sensor driver is probably half an hour,
>> then a bit of back and forth with the github checks / CLang formating and
>> can be live within the hour if your lucky, otherwise if there is feedback
>> or issues then about 3-10days seems to be average. (If you add the
>> component first like this pull request, then it can be approved as a
>> in-development component, so you can test the driver you write straight
>> away).
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1356842023>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A4AJFEI3RZLBKKO344VNRMLWN5CZLANCNFSM6AAAAAARZYT7CY>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

tyeth commented:
Yeah that looks very similar to the 9600, better one might imagine, and as you say with the qwiic connectors so gonna work very easily with wippersnapper. 
 The learn guide linked from the product page is for the MC9600 so we can fairly safely assume the 9601 is compatible (manufacturers product page or datasheets can confirm).

tyeth commented:
@rob-vaughn need to add one more line to the json, 
```json
  "published": false,
```
unless @brentru is happy to just publish straight to live (`true`) instead of as a Dev component first.

I've added PR #409 which adds the component officially, but you can test it as a SI7021 instead of SHT20 immediately and it should work correctly (that's all my pull-request adds)

brentru commented:
> unless @brentru is happy to just publish straight to live (true) instead of as a Dev component first.

I will set it once that PR is released

rob-vaughn commented:
Hi Tyeth,

I've been out of commission for a while. Is this live and ready to use now?

Thanks!

On Mon, Feb 13, 2023, 12:37 p.m. Tyeth Gundry ***@***.***>
wrote:

> @rob-vaughn <https://github.com/rob-vaughn> need to add one more line to
> the json,
>
>   "published": false,
>
> unless @brentru <https://github.com/brentru> is happy to just publish
> straight to live (true) instead of as a Dev component first.
>
> I've added PR #409 which adds the component officially, but you can test
> it as a SI7021 instead of SHT20 immediately and it should work correctly
> (that's all my pull-request adds)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1428259273>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEOUVKPZLBOODCB2VFTWXJPNZANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

tyeth commented:
> Hi Tyeth, I've been out of commission for a while. Is this live and ready to use now? Thanks!

It is! (It should also work for any compatible sensors - I think the HTU21D and something else)

rob-vaughn commented:
Awesome! Thanks!

On Thu, Mar 23, 2023, 4:00 p.m. Tyeth Gundry ***@***.***>
wrote:

> Hi Tyeth, I've been out of commission for a while. Is this live and ready
> to use now? Thanks!
>
> It is! (It should also work for any compatible sensors - I think the
> HTU21D and something else)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Wippersnapper_Components/pull/97#issuecomment-1481740284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4AJFEIE326JEQIS2ZTWQY3W5SMU5ANCNFSM6AAAAAARZYT7CY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
